Makefile

```diff
@@ -35,19 +39,29 @@ help:
 	@echo
 	@echo 'Configuration options:'
 	@echo
-	@echo '  GITHUB_USER       Which GitHub user account to use for cloning? [default=anonymous]'
-	@echo '  MINIKUBE_DRIVER   Which Minikube driver to use? [default=kvm2]'
-	@echo '  MINIKUBE_PROFILE  Which Minikube profile to use? [default=minikube]'
-	@echo '  TIMECHECK         Checking frequency in seconds when bringing cluster up and down? [default=5]'
-	@echo '  TIMEOUT           Maximum timeout to wait when bringing cluster up and down? [default=300]'
-	@echo '  VENV_NAME         Which Python virtual environment name to use? [default=reana]'
-	@echo '  DEMO              Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  GITHUB_USER         Which GitHub user account to use for cloning? [default=anonymous]'
+	@echo '  MINIKUBE_DRIVER     Which Minikube driver to use? [default=kvm2]'
+	@echo '  MINIKUBE_PROFILE    Which Minikube profile to use? [default=minikube]'
+	@echo '  TIMECHECK           Checking frequency in seconds when bringing cluster up and down? [default=5]'
+	@echo '  TIMEOUT             Maximum timeout to wait when bringing cluster up and down? [default=300]'
+	@echo '  VENV_NAME           Which Python virtual environment name to use? [default=reana]'
+	@echo '  DEMO                Which demo example to run? [e.g. reana-demo-helloworld; default=several runable examples]'
+	@echo '  CLUSTER_CONFIG      REANA Cluster environment mode [dev]'
```

Comment:
We don't need to expose two variables, `CLUSTER_CONFIG` and `DEBUG`, to the user, if one is derived from the other... Perhaps one is enough, or else let's expose the two but then let each has it own independent default that will be clearly visible in help.